src/launchpad/kafka.py

```diff
@@ -111,6 +111,7 @@ def create_kafka_consumer() -> LaunchpadKafkaConsumer:
         "enable.auto.commit": False,
         "enable.auto.offset.store": False,
         "security.protocol": config.security_protocol,
+        "session.timeout.ms": 300000,  # 5 minutes - how long before Kafka considers consumer dead. Add enough time for worst case rebalance
```

Comment:
Should this be routed through our `KafkaConfig`?

Reply:
I was on the fence since this value was based on the rebalancing timeout, but I'll revisit and potentially move it after confirming with streaming infra whether this is a good solution